tools/ci/retry.ps1

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env pwsh
+
+if ($args.Length -lt 1) {
+    Write-Host "Usage: $($MyInvocation.InvocationName) <command> [arguments]"
+    Write-Host "The script will execute the provided commands and retry in case of failures"
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏ looks like it only takes a single command
```suggestion
    Write-Host "The script will execute the provided command and retry in case of failures"
```